README.adoc

```diff
@@ -90,6 +90,14 @@ For pull requests, each rule can be triggered by:
 
 * `directories` - if any file in the commits of the pull requests match, trigger the rule. This is not a regexp (it uses `startsWith`) but glob type expression are supported too `extensions/test/**`.
 
+If no rule is triggered based on directories, or if rules are triggered but they all specify `allowSecondPass: true`,
+a second pass will be executed; in that second pass, rules can be triggered by:
+
+* `title` - if the title matches this regular expression (case insensitively), trigger the rule
+* `body` - if the body (i.e. description) matches this regular expression (case insensitively), trigger the rule
+* `titleBody` - if either the title or the body (i.e. description) match this regular expression (case insensitively), trigger the rule
+* `expression` - allows to write a Jakarta EL expression testing `title`, `body` or `titleBody`. Be careful when writing expressions, better ping `@gsmet` in the pull request when creating/updating an expression.
```

Comment:
@gsmet I'm not sure how to document this without introducing the concept of "second pass". 

We can give another name to that "second pass", of course, if you have one. But I don't think switching to a list of "imprecise labels" will eliminate the problem, since we'll still have to state somewhere that we apply rules a second time when all labels are imprecise.

Reply:
I'm not exactly excited about it but let's get it in as it's useful and we can tweak things later.